stockflux-securities-master/src/services/SecuritiesService.js

```diff
@@ -1,3 +1,10 @@
+export class ValidationError extends Error {
```

Comment:
Maybe it would be worth moving this into it's own file?

Reply:
Yep, done now